zrml/prediction-markets/src/mock.rs

```diff
@@ -303,6 +305,7 @@ impl pallet_treasury::Config for Runtime {
     type ProposalBondMaximum = ();
     type RejectOrigin = EnsureSignedBy<Sudo, AccountIdTest>;
     type SpendFunds = ();
+    type SpendOrigin = NeverEnsureOrigin<u128>;
```

Comment:
```suggestion
    type SpendOrigin = NeverEnsureOrigin<Balance>;
```

https://github.com/open-web3-stack/open-runtime-module-library/blob/02b1462a0e0e7bc06b3cdd6789c0e108f8a2b527/tokens/src/mock.rs#L133

https://paritytech.github.io/substrate/master/src/frame_support/traits/dispatch.rs.html#62-73

```rust
/// The origin required for approving spends from the treasury outside of the proposal
/// process. The `Success` value is the maximum amount that this origin is allowed to
/// spend at a time.
type SpendOrigin: EnsureOrigin<Self::Origin, Success = BalanceOf<Self, I>>;
```
Interesting: So we only allow spends of the treasury through the proposal process. Okay